modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointReadLockTimeoutException.java

```diff
@@ -32,6 +33,6 @@ public class CheckpointReadLockTimeoutException extends IgniteInternalException
      * @param msg Error message.
      */
     CheckpointReadLockTimeoutException(String msg) {
-        super(msg);
+        super(SYSTEM_CRITICAL_OPERATION_TIMEOUT_ERR, msg);
```

Comment:
Not sure about that, to be honest, but better safe than sorry, right?

Reply:
Yep, I think that the more concrete the error code the better.